bokeh/server/views/backbone.py

```diff
@@ -286,13 +286,15 @@ def update(docid, typename, id):
 @handle_auth_error
 def delete(docid, typename, id):
     #I don't think this works right now
+    obj = 'No this does not work, because obj is not defined, should it be an arg?'
```

Comment:
From looking at the surrounding functions, I could easily fix most bugs in this module, except this one. I don't know where `obj` is supposed to come from.